src/V3TraceDecl.cpp

```diff
@@ -655,11 +653,11 @@ class TraceDeclVisitor final : public VNVisitor {
 
         addToSubFunc(new AstTracePopPrefix{flp});
     }
-    // NOCOMMIT -- how to handle VL_* macro'ed types?
+    // NOCOMMIT  -- how to handle VL_* macro'ed types?
```

Comment:
I'm currently skipping module IO because the C++ types are these `VL*` macros (or rather the types they create) instead of what `vlArgType` says.  Is there a way to cast from one to the other?  I guess I could construct a `VlUnpacked` (or whatever) from the raw `WData` (or whatever), but then I'm be creating unnecessary intermediate assigments.

Reply:
Module I/O should only happen in the wrapper and there should be "normal" signals created the level below and that is what gets traced.  If the golden .out tests pass still (which they did) you should be good.